addons/website/static/src/snippets/s_social_media/options.js

```diff
@@ -0,0 +1,208 @@
+/** @odoo-module **/
+
+import options from 'web_editor.snippets.options';
+import {websiteDomain} from 'website.utils';
+
+options.registry.SocialMedia = options.Class.extend({
+    /**
+     * @override
+     */
+    async willStart() {
+        this._super(...arguments);
+        const websiteId = websiteDomain(this)[2][2];
```

Comment:
No :) just get what you need, the websiteId, not the whole domain to magically extract the id :p 
```suggestion
        let websiteId;
        this.trigger_up('context_get', {
            callback: function (ctx) {
                websiteId = ctx['website_id'];
            },
        });
```